### PR TITLE
added README along with inheritable rubocop configuration

### PR DESCRIPTION
**Pull request by ConnorSheremeta**
https://github.com/ualbertalib/library_applications_development/issues/100

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
...
\ No newline at end of file
+# Library Applications Development Inclusive Language
+University of Alberta Library Applications Development team is opting to use inclusive language instead of problematic terms in our team's projects and documentation through RuboCop's (https://github.com/rubocop/rubocop) Naming/InclusiveLanguage cop.
+
+Utilized the Edmonton Inclusive Language Guide along with the Android Open Source Project Respectful Code and Inclusive Documentation as a guide and reference for terms that are flagged as problematic and their suggested replacements:
+
+https://www.edmonton.ca/public-files/assets/document?path=PDF/InclusiveLanguageGuide2019.pdf
+https://source.android.com/docs/setup/contribute/respectful-code
+https://developers.google.com/style/inclusive-documentation#ableist-language
+https://developers.google.com/style/word-list
```
Comment (jefferya) [inline README.md]: Can these be added as a Markdown list to enhance readability?

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
-# library_applications_development_inclusive_language
\ No newline at end of file
+# Library Applications Development Inclusive Language
+University of Alberta Library Applications Development team is opting to use inclusive language instead of problematic terms in our team's projects and documentation through RuboCop's (https://github.com/rubocop/rubocop) Naming/InclusiveLanguage cop.
```
Comment (jefferya) [inline README.md]: A clarification on the use of language. Should the phrase "is opting to use" use stronger language, such as, "commits to use? 

A second clarification, is RuboCop Naming/InclusiveLangauge cop useable in non-Ruby repositories? If no, could we rephrase "through RuboCop's..." to something line "with the help of automated tools such as RuboCop's..."?

File: README.md [hunk 3]
```diff
@@ -1 +1,17 @@
...
+
+Utilized the Edmonton Inclusive Language Guide along with the Android Open Source Project Respectful Code and Inclusive Documentation as a guide and reference for terms that are flagged as problematic and their suggested replacements:
+
+https://www.edmonton.ca/public-files/assets/document?path=PDF/InclusiveLanguageGuide2019.pdf
+https://source.android.com/docs/setup/contribute/respectful-code
+https://developers.google.com/style/inclusive-documentation#ableist-language
+https://developers.google.com/style/word-list
+
+# How to use this in a project
```
Comment (jefferya) [inline README.md]: Should this be a secondary heading?

**Review by jefferya — COMMENTED** [review]
Thanks for the great references! A couple small clarifications and formatting enhancements.

File: README.md (hunk 2 above)
Comment (ConnorSheremeta) [reply README.md]: Those two language changes make sense to me :+1:

**Review by jefferya — APPROVED** [review]
LGTM!

**Review by jefferya — APPROVED** [review]
LGTM!

**Review by jefferya — APPROVED** [review]

**Review by jefferya — APPROVED** [review]

**Pull request merged**